content/post/github_topic_modeling_habrahabr.md

```diff
@@ -1,4 +1,13 @@
-![word cloud](http://blog.sourced.tech/post/github_topic_modeling/wordcloud.png)
+---
+author: vadim
```

Comment:
Yep, this is what I was talking about - we do not want to show this old russian post on habrahabr